vmware-event-router/internal/processor/aws_event_bridge.go

```diff
@@ -171,102 +178,70 @@ func NewAWSEventBridgeProcessor(ctx context.Context, cfg connection.Config, sour
 	return &eventBridge, nil
 }
 
-// Process implements the stream processor interface TODO: handle
-// throttling/batching
-// https://docs.aws.amazon.com/eventbridge/latest/userguide/cloudwatch-limits-eventbridge.html#putevents-limits
-func (awsEventBridge *awsEventBridgeProcessor) Process(moref types.ManagedObjectReference, baseEvent []types.BaseEvent) error {
-	batchInput, err := awsEventBridge.createPutEventsInput(baseEvent)
-	if err != nil {
-		errMsg := fmt.Errorf("could not create PutEventsInput for event(s): %v", err)
-		awsEventBridge.Println(errMsg)
-		return processorError(ProviderAWS, errMsg)
-	}
-
-	// nothing to send
-	if len(batchInput) == 0 {
-		return nil
+// Process implements the stream processor interface
+func (awsEventBridge *awsEventBridgeProcessor) Process(ce cloudevents.Event) error {
+	if awsEventBridge.verbose {
+		awsEventBridge.Printf("processing event (ID %s): %v", ce.ID(), ce)
 	}
 
-	for idx, input := range batchInput {
```

Comment:
Does moving to cloudevents mean we no longer process every event as is(stacking it in the processor)?

Reply:
Relying on event bridge to handle this for us?

Reply:
Not sure I understand the question. Is it about batching (previous version) vs non-batching (this PR)?

Reply:
I am sorry my comment was ambiguous. But in general yeah, my question was about removing the batching. But I saw:
```
Batching, e.g. as was in
EventBridge, could be re-implemented at a later stage (see note on state
management below)
.```
So I guess this answers my question